modules/page-header/css/README.md

```diff
@@ -39,20 +39,13 @@ Add your `node_modules` directory to your SASS `includePaths`. You will then be
 </header>
 ```
 
-### Marketing Contexts
-
-If you would like to use it in a marketing context with a fixed width and padding that matches other
-marketing content, simply replace `.wdc-page-header-container` with the
-[marketing container](../canvas-kit-css-marketing/#container) component
-(`.wdc-marketing-container`).
-
-**Example:**
+### With Cap Width
 
 ```html
 <header class="wdc-page-header">
-  <div class="wdc-marketing-container">
+  <div class="wdc-page-header-container wdc-page-header-cap-width">
```

Comment:
The class `wdc-page-header-container` isn't actually used for anything here. This div isn't even needed unless you're capping the width. What do you think about removing it, and updating the documentation to say something like "Add a wrapping div with `.wdc-page-header-cap-width` around the content if you'd like to cap the width at a maximum value"?

In addition, we should probably have a default story, and one with a custom breakpoint and cap width.

Reply:
@anicholls `.wdc-page-header-container` adds the display flex properties to align everything within the container.

Reply:
Whoops! Missed that line. NP, let's leave it 👍